### ISSUE TITLE
Añadir las pruebas métricas de interes para evaluar el resultado

### ISSUE THREAD

**Issue by NuriaVP** [NuriaVP/Transfer-Learning-EDM#28]
Como es una clasificación binaria (inicialmente descompensada) vamos a añadir empleando los resultados de la predicción las siguientes métricas: AUC y f-score.

**Comment by NuriaVP** [comment]
Fuentes de información sobre como aplicar en keras y sklearn estas medidas:

A) **Curva AUC-ROC**

https://runebook.dev/es/docs/tensorflow/keras/metrics/auc

B) **F-1 score**

https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html

**Comment by NuriaVP** [comment]
Los documentos que por ahora tienen resultados aceptables son "3 PRUEBA VGG16", "PRUEBA VGG19",  "VGG16 con class_weight" y "VGG16 con preprocesamiento gaussiano"; así que vamos a utilizar ambas métricas en las pruebas de estos archivos.

**Comment by NuriaVP** [comment]
Tarea completada en el commit: e49dc683157164a9e6a5cc200eda67856be66f0e

En esta tarea hemos añadido a las pruebas que parecía que ofrecían buenos resultados las métricas de evaluación más adecuadas para esta clasificación binaria. He modificado el cuaderno '3 PRUEBA VGG16', 'PRUEBA VGG19' y 'VGG16 con preprocesamiento gaussiano. De hecho en el último también he añadido entrenamientos con VGG19 junto con class_weight_compute y preprocesamiento gaussiano. Y en todos estos archivos he incluido como métricas de importancia: test loss, test accuracy, roc score, precision, recall y f1 score. Los resultados en muchos casos parecen buenos, y en tareas posteriores haré una tabla resumen con todos ellos.
